src/v/raft/consensus.cc

```diff
@@ -338,6 +338,12 @@ consensus::success_reply consensus::update_follower_index(
         // current node may change it.
         return success_reply::yes;
     }
+
+    // do not process follower busy replies further
+    if (r.value().result == reply_result::follower_busy) {
+        return success_reply::no;
+    }
```

Comment:
we already check it a few lines below, remove that check?

Reply:
done